package.json

```diff
@@ -125,44 +37,79 @@
     "bignumber.js": "8.1.1",
     "camelcase": "5.3.0",
     "copy-webpack-plugin": "5.0.4",
+    "core-js": "^2.6.5",
     "crypto-js": "3.1.9-1",
     "js-beautify": "1.9.1",
+    "lodash.isempty": "^4.4.0",
+    "lodash.orderby": "^4.6.0",
+    "lodash.uniqby": "^4.7.0",
     "moment": "2.24.0",
```

Comment:
would be great to import moment functions instead of the whole library - it's huge.

Reply:
We could use date-fns? https://date-fns.org/

It's a more functional design that doesn't mutate state, and allegedly smaller.

Reply:
whatever is easiest and accomplishes the goal!

Reply:
Not for this PR. Happy to convert after.